Doc/whatsnew/3.8.rst

```diff
@@ -405,17 +405,17 @@ Other Language Changes
 
   (Contributed by Jonathan Eunice and Serhiy Storchaka in :issue:`30688`.)
 
-* Dict and dictviews are now iterable in reversed insertion order using
-  :func:`reversed`. (Contributed by Rémi Lapeyre in :issue:`33462`.)
+* :class:`dict` and dictionary views are now iterable in reversed insertion
```

Comment:
I think there is a glossary item for dict viewe (can create a reference with the `term` role)

Reply:
Yep, looks like there is one https://docs.python.org/3/glossary.html#term-dictionary-view.
```suggestion
* :class:`dict` and :term:`dictionary view`s are now iterable in reversed insertion
```

Reply:
Looks like that caused it to go over the 80 char limit at 83 chars. I'll adjust the wrapping.

Reply:
ReST trick: to avoid having bits dangling out of markup like the plural `s`, there is a way to control the display text separate from the reference:

```rst
and :term:`dict views <dictionary view>`
```

Reply:
Ah I see you found that! :slightly_smiling_face:

Reply:
Yep, I vaguely remember reading about it in http://docutils.sourceforge.net/rst.html and used it before for explicit links. So, I tried it out on my local copy of `3.8.rst`, did `make html`, and it worked! (: